tgui/packages/tgui/interfaces/AirlockElectronics.js

```diff
@@ -85,6 +86,24 @@ export const AirlockElectronics = (props, context) => {
             })
           }
         />
+        <LabeledList>
+          <LabeledList.Item label="Airlock Speed">
+            <NumberInput
+              animate
+              unit="seconds"
+              step={0.5}
+              stepPixelSize={15}
+              minValue={2}
+              maxValue={30}
+              value={close_speed / 10}
+              onDrag={(e, value) =>
+                act('close_speed', {
+                  adjust: value - close_speed / 10,
+                })
+              }
+            />
+          </LabeledList.Item>
```

Comment:
maybe move this up to the other labeled list? up to you, I guess I'd like a screenshot to see how it looks, if you could get one?

Reply:
At a later date i'd like to add a toggle button for fast speed (and interface it with the wires), but right now it just hangs out under the stuff. Not the prettiest ui but it doesn't look too out of place.